src/components/media-player/media-range-selector.yml

```diff
@@ -4,9 +4,21 @@ modalTitle:
 playbackRange:
   en: "Playback range: from {from, select, start {the beginning} other {timecode {from}}} to {to, select, end {the end} other {timecode {to}}}"
   de: "Abspielbereich: von {from, select, start {Anfang} other {Time-Code {from}}} bis {to, select, end {Ende} other {Time-Code {to}}}"
+selectRangeLabel:
+  en: Set current playback timecode
+  de: Setz den aktuellen Wiedergabe Time-Code
+setAsStart:
+  en: as start time
+  de: als Anfangszeit
+setAsEnd:
+  en: as end time
+  de: als Endezeit
 loadingMessage:
   en: Determining media information ...
   de: Medieninformationen werden ermittelt ...
 errorMessage:
   en: An error occurred while retrieving the media file
   de: Beim Laden der Mediendatei ist ein Fehler aufgetreten
+setMaxRange:
+  en: Set maximum range
+  de: Maximaler Abspielbereich setzen
```

Comment:
I'd suggest:

"Reset to maximum range" / "Auf maximalen Bereich zurücksetzen"

Watcha think?

Reply:
I was initially thinking the same, but thought it's just very long. Can do it though, it will wrap on mobile.

Reply:
Set as discussed.

Reply:
:+1: